iib/workers/tasks/build_merge_index_image.py

```diff
@@ -200,8 +200,8 @@ def handle_merge_request(
             raise IIBError(err_msg)
 
         set_request_state(request_id, 'in_progress', 'Getting bundles present in the index images')
-        log.info('Getting bundles present in the source index image')
         with set_registry_token(overwrite_target_index_token, source_from_index):
+            log.info('Getting bundles present in the source index image')
```

Comment:
Is this change required?

Reply:
Probably should be a separate commit. fwiw,  I think it wouldn't matter if we bring it 2 lines below or keep it as is :)

Reply:
removed